### DISCUSSION TITLE
Update process

### DISCUSSION THREAD

**Discussion by chadamplifiedit** [microsoft/PubSec-Info-Assistant#590] [General]
What is the process to update the Information Assistant as new features and functionality are released?

**Comment by dayland** [comment]
In most cases, if you have forked the repo you simply need to:
1. Pull latest from main branch into your fork and resolve any merge conflicts. 
2. Check your `local.env` file against `local.env.example` as sometimes parameters are changed or added (These will be called out in Release Notes)
3. Rerun `make deploy`

We aim to try to main upgrades as seamless as possible. In some cases there are breaking changes that will require special steps to be performed. For example, in our upcoming v1.1 release, there will be extra steps required in the upgrade process as we transition from BICEP to terraform. Any special steps will be noted in the Release Notes and will come with extended markdown documentation on the process.

**Reply by rayxfelipe** [reply]
So, are the steps as follows?
1. Login to the existing codespace instance
2. In the terminal, run "git pull origin main"
3. Then the usual per instruction -- az login.... make deploy.

Is this correct? Thanks.

**Reply by dayland** [reply]
Correct.
